### PR TITLE
windows build

### PR DESCRIPTION
**Pull request by mingbowan**
based on https://github.com/pytorch/pytorch/pull/28677

### CODE REVIEW

File: .circleci/verbatim-sources/pytorch-job-specs.yml [hunk 1]
```diff
@@ -138,3 +138,64 @@ jobs:
...
+      SCCACHE_BUCKET: "ossci-compiler-cache"
+      CUDA_VERSION: "10"
+      USE_CUDA: "1"
+      TORCH_CUDA_ARCH_LIST: "7.5"
+    steps:
+      - checkout
+      - run:
+          name: Install Cuda
+          command: |
+            curl -kLO https://ossci-windows.s3.amazonaws.com/cuda_10.1.243_426.00_win10.exe
```
Comment (peterjc123) [inline .circleci/verbatim-sources/pytorch-job-specs.yml]: add `--retry N` here?

File: .circleci/verbatim-sources/pytorch-job-specs.yml [hunk 2]
```diff
@@ -138,3 +138,64 @@ jobs:
...
+            ./cuda_10.1.243_426.00_win10.exe -s
+            rm -f ./cuda_10.1.243_426.00_win10.exe
+      - run:
+          name: Install Cudnn
+          command : |
+            cd c:/
+            choco install -y 7zip curl
+            choco install cmake --installargs 'ADD_CMAKE_TO_PATH=User'
+            curl -O https://ossci-windows.s3.amazonaws.com/cudnn-10.1-windows10-x64-v7.6.4.38.zip
```
Comment (peterjc123) [inline .circleci/verbatim-sources/pytorch-job-specs.yml]: add `--retry N` here?

**Review by peterjc123 — APPROVED** [review]
Great!

File: torch/csrc/jit/passes/quantization.cpp [hunk 3]
```diff
@@ -1235,13 +1235,13 @@ graph(%a_dequant, %w, %b, %w_scale, %w_zero_point, %w_axis, %w_dtype):
 graph(%a_dequant, %w, %b, %w_scale, %w_zero_point, %w_dtype, %stride, %padding, %dilation, %groups):
         %w_quant = aten::quantize_per_tensor(%w, %w_scale, %w_zero_point, %w_dtype)
         %packed_params = quantized::conv2d_prepack(%w_quant, %b, %stride, %padding, %dilation, %groups)
-        return (%packed_params))");
+        return (%packed_params) )");
```
Comment (ezyang) [inline torch/csrc/jit/passes/quantization.cpp]: This is one of those things where it's very helpful to have a comment explaining why there needs to be an extra space here.

**Review by ezyang — APPROVED** [review]
Great work!

**Pull request closed without merging**